Makefile

```diff
@@ -2,18 +2,26 @@ PYTHON ?= python3
 
 makefile_dir := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
-all: bblfsh/github/com/gogo/protobuf/gogoproto/gogo_pb2.py \
-     bblfsh/github/com/bblfsh/sdk/uast/generated_pb2.py \
-     bblfsh/github/com/bblfsh/sdk/protocol/generated_pb2_*.py \
-     bblfsh/github/__init__.py \
-     bblfsh/github/com/__init__.py \
-     bblfsh/github/com/gogo/__init__.py \
-     bblfsh/github/com/gogo/protobuf/__init__.py \
-     bblfsh/github/com/gogo/protobuf/gogoproto/__init__.py \
-     bblfsh/github/com/bblfsh/__init__.py \
-     bblfsh/github/com/bblfsh/sdk/__init__.py \
-     bblfsh/github/com/bblfsh/sdk/uast/__init__.py \
-     bblfsh/github/com/bblfsh/sdk/protocol/__init__.py
+all: bblfsh/pyuast.py \
+	bblfsh/github/com/gogo/protobuf/gogoproto/gogo_pb2.py \
+	bblfsh/github/com/bblfsh/sdk/uast/generated_pb2.py \
+	bblfsh/github/com/bblfsh/sdk/protocol/generated_pb2_*.py \
+	bblfsh/github/__init__.py \
+	bblfsh/github/com/__init__.py \
+	bblfsh/github/com/gogo/__init__.py \
+	bblfsh/github/com/gogo/protobuf/__init__.py \
+	bblfsh/github/com/gogo/protobuf/gogoproto/__init__.py \
+	bblfsh/github/com/bblfsh/__init__.py \
+	bblfsh/github/com/bblfsh/sdk/__init__.py \
+	bblfsh/github/com/bblfsh/sdk/uast/__init__.py \
+	bblfsh/github/com/bblfsh/sdk/protocol/__init__.py
+
+libuast:
+	git clone git@github.com:manucorporat/libuast.git libuast
+	cd libuast && cmake . && make && make install
+
+bblfsh/pyuast.py: libuast
+	cd bblfsh && swig -c++ -python -I/usr/local/include uast.i
```

Comment:
So `swig` is needed, at least for `make`. Since the `.i` and `pyuast.py` files are included, couldn't we remove the `all:` target dependency on this one? (but keeping this in the Makefile just in case we want to regenerate `pyuast.py`).